doc/ref_arch/kubernetes/chapters/chapter02.md

```diff
@@ -306,8 +306,6 @@ The requirements in this section are to be delivered in addition to those in [se
 | `req.inf.ntw.08` | Infrastructure | Network | The Architecture **must**  provide a default network which implements the Kubernetes network model. | |
 | `req.inf.ntw.09` | Infrastructure | Network | The networking solution **must not** interfere with or cause interference to any interface or network it does not own. | |
 | `req.inf.ntw.10` | Infrastructure | Network | The Architecture **must** support cluster wide coordination of IP address assignment. | |
-| `req.inf.ntw.12` | Infrastructure | Network | The networking solution to support multiple connection points (e.cap.004 - please see [Section 2.2.1](#2.2.1)) must support any Kubernetes conformant CNI plugin to be incorporated as a networking solution. | |
```

Comment:
What is the reason to remove this?

Reply:
I don't think this is correct (stops the use of NSM, for example) and even if we agree it is, I think it is a specification not an input requirement.

Reply:
I've added `ra2.ntw.007` in https://github.com/cntt-n/CNTT/pull/2121/commits/82ba2060adaadb5b54aa8c395c02b4d25b74c1ed to capture this specification.  My feeling is CNI is part of the spec, not an input requirement.